credentials/urls.py

```diff
@@ -34,11 +36,21 @@
 admin.site.site_header = _("Credentials Administration")
 admin.site.site_title = admin.site.site_header
 
+schema_view = get_schema_view(
+   openapi.Info(
+      title="Credentials API",
+      default_version='v1',
+      description="Credentials API docs",
+   ),
+   public=False,
```

Comment:
Needs 4 spaces, and then another 4 spaces of indentation:

```suggestion
    openapi.Info(
        title="Credentials API",
        default_version="v1",
        description="Credentials API docs",
    ),
    public=False,
    permission_classes=[permissions.AllowAny],
```